mm/tlsf/mm_tlsf.c

```diff
@@ -450,7 +448,7 @@ static void memdump_handler(FAR void *ptr, size_t size, int used,
 #endif
         }
     }
-  else if (dump->pid <= MM_BACKTRACE_FREE_PID)
+  else if (dump->pid == PID_MM_FREE)
```

Comment:
just a note that logic is changed here. what is the intention?

Reply:
Yes, the origin code is wrong